transform/transform_test.go

```diff
@@ -177,6 +177,27 @@ func TestTransformInstructions(t *testing.T) {
 			in:   testRaw,
 			want: "outout2",
 		},
+		{
```

Comment:
Nice to add it here, I think it is also sufficiently different that you should add a test case to TestTransformUnmarshal